lib/util.js

```diff
@@ -44,7 +44,7 @@ function getErrorsFromSchema(initialValues, schema, errors = {}) {
     switch (true) {
       case schema[key].type === 'object' && !isEmpty(schema[key].fields): {
         errors[key] = getErrorsFromSchema(
-          initialValues[key],
+          isEmpty(initialValues) ? undefined : initialValues[key],
```

Comment:
can we use null here? `undefined` can mean a coding mistake. can we also add a test case for this? we'll need to craft a nested schema similar to the trusted contact form schema

https://github.com/tjinauyeung/svelte-forms-lib/blob/master/test/specs/library.spec.js#L73

```suggestion
          isEmpty(initialValues) ? null : initialValues[key],
```

Reply:
yes i originally had null here but it threw errors in test suite saying to use undefined instead of null